src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesSynchronisationIntTest.kt

```diff
@@ -1155,6 +1125,23 @@ fun caseNoteEvent(
 }
 """.trimIndent()
 
+fun caseNoteDeleteEvent(
+  bookingId: Long = BOOKING_ID,
+  caseNoteId: Long = NOMIS_CASE_NOTE_ID,
+) = """{
+    "MessageId": "ae06c49e-1f41-4b9f-b2f2-dcca610d02cd", "Type": "Notification", "Timestamp": "2019-10-21T14:01:18.500Z", 
+    "Message": "{\"eventType\":\"OFFENDER_CASE_NOTES-DELETED\",\"eventDatetime\":\"2024-07-10T15:00:25.489964\",\"bookingId\": \"$bookingId\",\"caseNoteId\": \"$caseNoteId\",\"caseNoteType\":\"PRISON\",\"caseNoteSubType\":\"RELEASE\",\"recordDeleted\":true }",
```

Comment:
would we still have a bookingId come through? The comment above implies the bookingId doesn't exist too.  Not sure if it actually matters here, anyway.